CHANGELOG.rst

```diff
@@ -10,6 +10,16 @@ Changelog
 v.2.9.0 TBA
 ==================
 
+Minor changes:
+
+ * For navl schemas, the 'default' validator no longer applies the default when
+   the value is False, (), [] or {} (#4448)
```

Comment:
please remove `()` (shouldn't appear as part of a json payload) and add `0` to this list.